src/App.tsx

```diff
@@ -14,11 +16,82 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type Props = {};
+
+type State = {
+  selectedGood: string;
+};
+
+class App extends React.Component<Props, State> {
+  state: State = {
+    selectedGood: 'Jam',
+  };
```

Comment:
if you not use `pros`, delete it
```suggestion

type State = {
  selectedGood: string;
};

class App extends React.Component<{}, State> {
  state: State = {
    selectedGood: 'Jam',
  };
```